storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -7104,6 +7104,14 @@ static void xtrabackup_prepare_func(int argc, char **argv) {
   IORequest write_request(IORequest::WRITE);
 
   read_metadata();
+  if (strcmp(metadata_type_str, "log-applied") == 0 &&
+      !xtrabackup_incremental) {
+    xb::error() << "Backup is already prepared. The final incremental should "
+                   "be prepared without --apply-log-only. or if you restoring "
+                   "full backup, It should be only prepare without option "
+                   "--apply-log-only";
```

Comment:
I don't understand the intention of the second part of the message "or if you restoring.."

Reply:
PXB-2180 is fixed in 8.0.23. Is it possible to reproduce these assertions using PXB 8.0.23? (with the disable page cleaner patch)?